packages/core/src/Stores/Helpers/chart-markers.js

```diff
@@ -127,13 +130,20 @@ export const createTickMarkers = (contract_info, is_delayed_markers_update) => {
             tick.align_label = 'top'; // force exit spot label to be 'top' to avoid overlapping
             marker_config = createMarkerSpotExit(contract_info, tick, idx);
         }
-        if (is_smarttrader_contract && is_middle_spot) {
-            const spot_className = marker_config.content_config.spot_className;
-            marker_config.content_config.spot_className = `${spot_className} ${spot_className}--smarttrader-contract-middle`;
+
+        const spot_className = marker_config && marker_config.content_config.spot_className;
```

Comment:
is it supposed to be boolean or string? Looks like it'll return a boolean?

Reply:
It will return a string, it first checks for existence of `market_config` and then proceeds to assign `marker_config.content_config.spot_className` to the variable.

Reply:
if `marker_config === false`, then `spot_className` will be equal to `false` too.

Reply:
This condition is just to make sure that it doesn't break the whole app if it's trying to access it. If it's false or undefined doesn't matter imho, as it doesn't impact anyhow.